packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -15,10 +15,29 @@ export default class AppStore {
         this.api_helpers_store = null;
     }
 
+    error_for_non_eu_client = {
+        text: localize(
+            'Unfortunately, this trading platform is not available for EU Deriv account. Please switch to a non-EU account to continue trading.'
+        ),
+        title: localize('Deriv Bot is unavailable for this account'),
+        link: localize('Switch to another account'),
+    };
+
+    getErrorForEuClients = is_logged_in => {
+        return {
+            text: localize(' '),
```

Comment:
why are we localizing a space? 😅

Reply:
let me fix it 😳

Reply:
There was a text, but removed it and the localize remained by mistake. Removing it.

Reply:
fixed✅